fancyimpute/matrix_factorization.py

```diff
@@ -37,6 +37,7 @@ def __init__(
             self,
             rank=10,
             learning_rate=0.001,
+            epochs = 10000
```

Comment:
Please get rid of the spaces around = to conform to the rest of the code.

Reply:
fixed this